css/settings.css

```diff
@@ -78,4 +78,8 @@ label{
 
 .subtitle{
     font-weight: 300;
+}
+
+input:invalid {
```

Comment:
Instead of adding this rule I believe we can remove the border property from the `.app-settings input` rule which is the one that made it, so the border didn't turn red in the first place.

I believe the reason why it might have been added has been resolved with some changes to the CSS variables.

Reply:
Updated it here https://github.com/MetaProvide/adminly_core/pull/100